iphone/cli/commands/_buildModule.js

```diff
@@ -141,10 +141,11 @@ iOSModuleBuilder.prototype.initialize = function initialize() {
 	this.metaDataFile = path.join(this.projectDir, 'metadata.json');
 	this.manifestFile = path.join(this.projectDir, 'manifest');
 	this.templatesDir = path.join(this.platformPath, 'templates');
+	this.hooksDir = path.join(this.platformPath, 'hooks');
 	this.assetsTemplateFile = path.join(this.templatesDir, 'module', 'default', 'template', 'iphone', 'Classes', '{{ModuleIdAsIdentifier}}ModuleAssets.m.ejs');
 	this.universalBinaryDir = path.join(this.projectDir, 'build');
 
-	['assets', 'documentation', 'example', 'platform', 'Resources'].forEach(function (folder) {
+	['assets', 'documentation', 'example', 'platform', 'Resources', 'hooks'].forEach(function (folder) {
```

Comment:
If you add `'hooks'` here, you will override `this.hooksDir` above, so best not to do that.